chromium_src/components/search_engines/brave_template_url_prepopulate_data_unittest.cc

```diff
@@ -156,28 +156,10 @@ TEST_F(BraveTemplateURLPrepopulateDataTest, DefaultSearchProvidersForUSA) {
   CheckForCountry('U', 'S', "Google");
 }
 
-TEST_F(BraveTemplateURLPrepopulateDataTest, DefaultSearchProvidersForGermany) {
-  CheckForCountry('D', 'E', "DuckDuckGo");
-}
-
 TEST_F(BraveTemplateURLPrepopulateDataTest, DefaultSearchProvidersForFrance) {
   CheckForCountry('F', 'R', "Qwant");
 }
 
-TEST_F(BraveTemplateURLPrepopulateDataTest,
-       DefaultSearchProvidersForAustralia) {
-  CheckForCountry('A', 'U', "DuckDuckGo");
-}
-
-TEST_F(BraveTemplateURLPrepopulateDataTest,
-       DefaultSearchProvidersForNewZealand) {
-  CheckForCountry('N', 'Z', "DuckDuckGo");
-}
-
-TEST_F(BraveTemplateURLPrepopulateDataTest, DefaultSearchProvidersForIreland) {
-  CheckForCountry('I', 'E', "DuckDuckGo");
-}
-
```

Comment:
I understand that the reason these tests are removed is because these 4 countries now default back to Google as the search engine, but shouldn't we change the expectation instead of removing them, so that we don't lose test coverage?

Reply:
That's a good point; and we definitely can do that 😄

Reply:
Updated 👍